Joshua/Parkour.cs

```diff
@@ -0,0 +1,118 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+public class Parkour : MonoBehaviour
+{
+    Transform chController;
+    private bool inside = false;
+    [SerializeField]
+    private float heightFactor = 8.0f;
+    private KeyCode shift = KeyCode.LeftShift;
+    private string parkourObj;
+    [SerializeField]
+    private float speed = 14.0f;
+    [SerializeField]
+    private float slideCooldown = 1.5f;
+    [SerializeField]
+    private float slideCooldownTime = 1f;
+
+
+    void Start(){
+
+    }
+
+
+    //check for collision with gameobject with tag
+    void OnTriggerEnter(Collider Col){
```

Comment:
Op line 7 moet het "private Transform chController" zijn.
Ik zou op line 27 "Col" in camelCasing doen omdat het een variable is!
Al je functies moeten ook private of public worden!
Als je de Start() functie niet gebruik kan je hem net zo goed weg laten.

Verder is alles wel genoemd denk ik. 
Prima code bud! *high-fives*